docs/api/js/zksync2-js/providers.md

```diff
@@ -146,7 +146,7 @@ console.log(`Gas for token approval tx: ${gasTokenApprove}`);
 ```ts
 import { Provider, types } from "zksync2-js";
 
-const provider = Provider.getDefaultProvider(types.Network.Goerli);
+const provider = Provider.getDefaultProvider(types.Network.Sepolia);
 const tokenAddress = "0xCd9BDa1d0FC539043D4C80103bdF4f9cb108931B"; // Crown token which can be minted for free
```

Comment:
Same here, token and paymaster is not deployed on sepolia

Reply:
This was updated with the new tokenAddress you created!